cumulusci/tasks/bulkdata/load.py

```diff
@@ -429,15 +431,15 @@ def _expand_mapping(self):
 
         for step in self.mapping.values():
             step["action"] = step.get("action", "insert")
-            if "lookups" in step and any(
-                ["after" in l for l in step["lookups"].values()]
+            if step.get("lookups") and any(
+                [l["after"] for l in step["lookups"].values() if l.get("after")]
             ):
                 # We have deferred/dependent lookups.
                 # Synthesize mapping steps for them.
 
                 sobject = step["sf_object"]
                 after_list = {
-                    l["after"] for l in step["lookups"].values() if "after" in l
+                    l["after"] for l in step["lookups"].values() if l.get("after")
```

Comment:
As above

Reply:
Not in this case. Because the list will end up with Nones in it.